rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -16,13 +16,15 @@ toc::[]
 [id="rosa-q3-2024_{context}"]
 === Q3 2024
 
+* **{hcp-title} multi-architecture cluster update.** {hcp-title-first} clusters created before 25 July, 2024 will migrate to a multi-architecture image on their next upgrade allowing you to use {AWS} Arm-based Graviton instance types for your workloads. For more information, see xref:../upgrading/rosa-hcp-upgrading.adoc#rosa-upgrade-options_rosa-hcp-upgrading[Upgrading ROSA with HCP clusters].
```

Comment:
@xJustin Are clusters created after this date already capable of using these instance types? May be worth clarifying here so folks who don't have to do anything don't have to click through.

Reply:
yes they should. Any cluster created after July 25th is using a multi architecture image and should be able to use Graviton instance types
It is written a few lines after on the release notes:
```
ROSA with HCP Graviton Arm-based instance types. You can now use Amazon Web Services (AWS) Arm-based Graviton instance types for your workloads in Red Hat OpenShift Service on AWS (ROSA) with hosted control planes (HCP) clusters created after 24 July, 2024, see [AWS Graviton Arm-based instance types](https://82455--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-hcp-instance-types#rosa-sdpolicy-aws-instance-types-graviton_rosa-hcp-instance-types).
```